packages/lit-dev-content/samples/tutorials/advanced-templating/tutorial.json

```diff
@@ -0,0 +1,17 @@
+{
+  "header": "Advanced Templating",
```

Comment:
Question about "draft"/"in progress" /"dummy" tutorials.

Thoughts around having a `draft` or `inProgress` key-val pair? Maybe it just prevents the in progress tutorials from being rendered on the main /tutorials/ page. Since these currently can only be accessed by link no action required.

Reply:
Theoretically you can just change the `category` to draft. The tutorials page gets all these and filters by category. If it's not in either Build or Learn, then it'll just not display them